src/components/extractionQc/ExtractionQcForm.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <div>
+    <traction-form class="text-left" @submit="onSubmit">
+      <traction-sub-section title="Which QC Results would you like to upload?" class="py-6">
+        <traction-select
+          v-model="usedBySelected"
+          :options="usedByOptions"
+          :state="Boolean(usedBySelected) ? true : null"
+          required
+        ></traction-select>
+      </traction-sub-section>
+      <traction-sub-section title="CSV File" class="py-6">
+        <traction-file
+          v-model="file"
+          :state="Boolean(file) ? true : null"
```

Comment:
simplifies to !!file

Reply:
Same as https://github.com/sanger/traction-ui/pull/979#discussion_r1049757454